app/models/manageiq/providers/ansible_tower/automation_manager/refresher.rb

```diff
@@ -1,3 +1,3 @@
-class ManageIQ::Providers::AnsibleTower::AutomationManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
+class ManageIQ::Providers::AnsibleTower::AutomationManager::Refresher < ManageIQ::Providers::BaseManager::ManagerRefresher
```

Comment:
@slemrmartin is this required?  I'm trying to remove ManagerRefresh in another PR

Reply:
@agrare it raises error when I'm refreshing `ConfigurationScriptSource` with Refresher. 
I think there is error in `collector_inventory_for_targets()`. In `ManagerRefresher`, it derives Collector class in `Inventory`, there it tries to do by itself (https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/base_manager/refresher.rb#L141). This construction doesn't work for TargetCollection. 
Then it calls `parse_targeted_inventory` with `collector == nil`.

Reply:
sorry, for `InventoryRefresh::Target` it doesn't work...it contantizes `ManageIQ::Providers::AnsibleTower::Inventory::Collector::Target` instead of `ManageIQ::Providers::AnsibleTower::Inventory::Collector::TargetCollection`